src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/Setup/useDefaultColumnConfig.js

```diff
@@ -35,7 +35,37 @@ const mainConfig: Array<MainColumnConfig> = [{
         mainProperty: true,
     }));
 
-const getMetaDataConfig = (attributes: Array<DataElement>, orgUnitId: ?string): Array<MetadataColumnConfig> =>
+const programStageMainConfig: Array<MainColumnConfig> = [{
+    id: 'status',
+    visible: false,
+    type: dataElementTypes.TEXT,
+    header: i18n.t('Event status'),
+    filterHidden: true,
+},
+{
+    id: 'occurredAt',
+    visible: false,
+    type: dataElementTypes.DATE,
+    header: i18n.t('Report date'),
```

Comment:
if the report date label has been configured in the maintenance app, we should display it here.